pkg/api/printer.go

```diff
@@ -0,0 +1,89 @@
+package api
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+	"text/tabwriter"
+)
+
+func (r *Request) PrintObj() string {
+	out, err := tabbedString(func(out io.Writer) error {
+		fmt.Fprintf(out, "Base Image:\t%s\n", r.BaseImage)
```

Comment:
In all our docs we call that the builder image, I'm aware of the var naming, still I'd say to keep with the Builder name.